packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.js

```diff
@@ -200,6 +200,7 @@ describe('ReactDOMFiberAsync', () => {
 
       // Should flush both updates now.
       jest.runAllTimers();
+      global.flushRequestAnimationFrameQueue();
```

Comment:
Since these tests use `requestIdleCallback`, we need to flush the `rAF` queue, but these seems weird to me because the environment is so mocked at this point that it's not really testing the actual browser behavior.

In other words, the test passes just because we manually flush in the correct order.

Reply:
requestIdleCallback is not an important aspect of these tests. You can rewrite them to startTransition